src/ResourceManager/Compute/Commands.Compute/AzureRM.Compute.psd1

```diff
@@ -59,10 +59,8 @@ RequiredAssemblies = '.\AutoMapper.dll', '.\Microsoft.Azure.Management.Compute.d
                '.\Microsoft.Azure.Management.Storage.dll', 
                '.\Microsoft.Data.Edm.dll', '.\Microsoft.Data.OData.dll', 
                '.\Microsoft.Data.Services.Client.dll', 
-               '.\Microsoft.WindowsAzure.Commands.Sync.dll', 
```

Comment:
Are these now part of the compute assembly on desktop?

Reply:
Desktop is now referencing the source files directly from the Sync project to build. This is the same strategy that NetStandard does.